vizro-core/docs/pages/user-guides/layouts.md

```diff
@@ -403,10 +413,329 @@ By default, the grid fits all charts/components on the screen. This can lead to
+                       In ultricies est ac mauris vehicula fermentum. Curabitur faucibus elementum lectus.
+                       Name ut ipsum tortor. Praesent ut nulla risus. Praesent in dignissim nulla.
+                   """
+                )
+                for i in range(12)
+            ],
+        )
+
+        dashboard = vm.Dashboard(pages=[page])
+        ```
+
+        <img src=https://py.cafe/logo.png alt="PyCafe logo" width="30"><b><a target="_blank" href="https://py.cafe/vizro-official/vizro-custom-card-styling">Run and edit this code in PyCafe</a></b>
+
+    === "Result"
+        [![FlexItemSizeCSS]][flexitemsizecss]
+
+### Combine Flex and Grid
+
+You can also combine the `Flex` and `Grid` layout to benefit from both. The `Grid` layout is ideal for defining the overall page structure, while the `Flex` layout enables flexibility within individual sections.
+
+For example, in the layout below, we use the `Grid` layout to arrange two charts at the top and a container with multiple cards at the bottom. Within this card container, we apply the `Flex` layout so that the cards automatically adjust their positioning based on the available space.
+
+!!! example "Grid with Flex container"
+    === "app.py"
+        ```{.python pycafe-link}
+        import vizro.models as vm
+        import vizro.plotly.express as px
+        from vizro import Vizro
+
+        tips = px.data.tips()
+
+        page = vm.Page(
+            title="Combine Flex and Grid",
+            layout=vm.Layout(grid=[[0, 1], [2, 2]]),
+            components=[
+                vm.Graph(figure=px.violin(tips, y="tip", x="day", color="day", box=True)),
+                vm.Graph(figure=px.histogram(tips, x="total_bill")),
+                vm.Container(
+                    title="Flexbox with Cards",
+                    layout=vm.Flex(direction="row", wrap=True),
+                    components=[
+                        vm.Card(
+                            text="""
+                                    # Lorem Ipsum
+
+                                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam sed elementum ligula, in pharetra velit.
+                                    In ultricies est ac mauris vehicula fermentum. Curabitur faucibus elementum lectus, vitae luctus libero fermentum.
+                                    Name ut ipsum tortor. Praesent ut nulla risus. Praesent in dignissim nulla. In quis blandit ipsum.
+                                """,
+                            extra={"style": {"width": "240px"}},
+                        )
+                        for i in range(6)
+                    ],
+                ),
+            ],
+        )
+
+        dashboard = vm.Dashboard(pages=[page])
+        Vizro().build(dashboard).run()
+        ```
+
+    === "Result"
+        [![FlexGridCombined]][flexgridcombined]
+
 ## Alternative layout approaches
```

Comment:
I think this section needs a slight rewording just to reflect the fact that grid isn't the only possible approach now.